packages/bpk-component-switch/src/BpkSwitch.js

```diff
@@ -0,0 +1,60 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2020 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* @flow strict */
+
+import PropTypes from 'prop-types';
+import React, { type Node } from 'react';
+import { cssModules } from 'bpk-react-utils';
+
+import STYLES from './BpkSwitch.scss';
+
+const getClassName = cssModules(STYLES);
+
+export type Props = {
+  label: Node,
+  className: ?string,
+};
+const BpkSwitch = (props: Props) => {
+  const { className, label, ...rest } = props;
+  const classNames = getClassName('bpk-switch', className);
+
+  return (
+    <label className={classNames}>
+      <input
+        type="checkbox"
+        className={getClassName('bpk-switch__checkbox')}
```

Comment:
Shouldn't this have `aria-label={label}`?

Reply:
Yes! Great catch 😅